german/schwimmen/trommeln/trommeln.md

```diff
@@ -0,0 +1,7 @@
+Der rythmische Trommelklang hat die gestrandeten angelockt. Sie tapsen durch den Sand, schlagen sich durch die Büsche während die dumpfen Klänge lauter werden. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.